### DISCUSSION TITLE
How to uninstall manually?

### DISCUSSION THREAD

**Discussion by RadenDimasErlangga21** [voicepaw/so-vits-svc-fork#714] [Q&A] [answered]
Where is the installation directory at

**Comment by upColor** [comment, marked as answer]
The folder: 
C:\Users\user name\AppData\Roaming\so-vits-svc-fork

Yeah, I have the same issue. I don't know how to uninstall.

I guess that uninstalling svc = removing virtual environment in Python. 
If this is true, then this might help:
https://stackoverflow.com/questions/11005457/how-do-i-remove-delete-a-virtualenv
